src/Tuura/Concept/Concat/Circuit.hs

```diff
@@ -58,8 +58,11 @@ type CircuitConcept a = Concept (State a) (Transition a) a
 consistency :: CircuitConcept a
 consistency = mempty
 
-initialise :: a -> Bool -> CircuitConcept a
-initialise a = initialConcept . after . Transition a
+initialise :: Eq a => a -> Bool -> CircuitConcept a
+initialise a v = initialConcept $ \s -> if s == a then Defined v else Undefined
+
+initials :: Eq a => [a] -> Bool -> CircuitConcept a
```

Comment:
Another way could be to have two functions `initialise0` and `initialise1` with type `Eq a => [a] -> CircuitConcept a`, so one can simply write `initialise0 [a, b, c] <> initialise1 [d]`.